beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -343,6 +351,19 @@ func (s *Service) validateBellatrixBeaconBlock(ctx context.Context, parentState
 	return nil
 }
 
+// Verifies the signature of the pending block with respect to the current head state.
+func (s *Service) verifyPendingBlockSignature(ctx context.Context, blk interfaces.ReadOnlySignedBeaconBlock, blkRoot [32]byte) (pubsub.ValidationResult, error) {
+	roState, err := s.cfg.chain.HeadStateReadOnly(ctx)
+	if err != nil {
+		return pubsub.ValidationIgnore, err
+	}
```

Comment:
Couldn't an edge case exist where the head state is insufficient to verify the block signature? Ie. Across hard fork boundary or during long forking times